crates/engine/src/ext.rs

```diff
@@ -343,10 +343,6 @@ impl Engine {
         super::hashing::keccak_256(input, output);
     }
 
-    pub fn now(&self, _output: &mut &mut [u8]) {
-        unimplemented!("off-chain environment does not yet support `now`");
-    }
-
```

Comment:
What happened here?

Reply:
Basically I just rename it from `now` to `block_timestamp`. We don't use `now` anywhere, just the latter.